src/controllers/share/share.controller.ts

```
@@ -0,0 +1,93 @@
+    if (difficulty < submission.difficulty) {
+      throw new UnauthorizedException('Invalid share - calculated difficulty does not match claimed difficulty');
+    }
+
+    // Store the share statistics
+    await this.clientStatisticsService.insert({
+      address: submission.address,
+      clientName: submission.worker,
+      sessionId: submission.sessionId,
+      time: new Date().getTime(),
+      shares: submission.difficulty,
+      acceptedCount: 1
```

ESLint

[error] Insert `,`

(prettier/prettier)